Resources/Prototypes/Entities/Objects/Specific/Xenoarchaeology/artifact_equipment.yml

```
@@ -124,7 +124,7 @@
     - 0,0,1,1
     whitelist:
       components:
-      - Artifact
+      - XenoArtifact # Frontier Artifact<XenoArtifact
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)